plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -122,7 +120,7 @@ private void launchDeployJob(IProject project, Credential credential)
                                               new DeployConsole.Factory());
 
     MessageConsoleStream outputStream = messageConsole.newMessageStream();
-    StandardDeployJobConfig config = getDeployJobConfig(project, credential,
+    StandardDeployJobConfig config = createDeployJobConfig(project, credential,
```

Comment:
I meant inlining `createDeployJobConfig()` here.  There are a lot of deploy-config objects being thrown around here and it's a bit confusing.  Would it make sense to rename this `StandardDeployJobConfig` as a _Parameterization_?  Or move this into the Job?